README.md

```diff
@@ -7,6 +7,10 @@ Miscellaneous functions for aiding analysis during LCLS experiments.
 
 see *Doxygen* documentation here: https://lcls-users.github.io/btx/
 
+## Git Flow convention
+
+As of October 19 2023, this repository uses the git flow convention. All required information can be found in this short and clear cheatsheet: http://dominhhai.github.io/git-flow-cheatsheet/index.html
+
```

Comment:
Shall I remove this from the README.md?

Reply:
Yes I think so.

Reply:
Agreed, we should remove this since we are using slightly different conventions with multiple repos, and we should clarify how this works to avoid confusion. That said I don't want that to be the reason the PR gets blocked, so lets just note that this needs to be revised.